spec/rails_helper.rb

```diff
@@ -50,7 +50,7 @@ def create_admin
 
   def create_rental
     let!(:rental_one) { Rental.create(user_id: User.last.id, days_rented: 5, total_price: 190, status: 'Completed', created_at: "2015-11-11 15:29:36") }
-    let!(:rental_two) { Rental.create(user_id: User.last.id, days_rented: 2, total_price: 28, status: 'Completed', created_at: "2015-11-11 15:29:36") }
+    let!(:rental_two) { Rental.create(user_id: User.last.id, days_rented: 2, total_price: 28, status: 'Ordered', created_at: "2015-11-11 15:29:36") }
```

Comment:
Line is too long. [149/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.